tests/tests_fabric/conftest.py

```diff
@@ -55,6 +55,7 @@ def restore_env_variables():
         "POPLAR_ENGINE_OPTIONS",  # set by IPUStrategy
         "CUDA_MODULE_LOADING",  # leaked since PyTorch 1.13
         "CRC32C_SW_MODE",  # set by tensorboardX
+        "CUBLAS_WORKSPACE_CONFIG",  # handled by the `reset_deterministic_algorithm` fixture below
```

Comment:
TODO: check if still needed

Reply:
Ah I see. You added it in https://github.com/Lightning-AI/lightning/pull/16899/files#diff-b2392d6bfca458f285f2e51e1e5c4a43a2f43e1c0e0eac15bd291d3662b6c87fR20.

The best way to fix this is to add an environment clear patch to the tests that call `make_deterministic`

Reply:
Nope, can't clear the env for standalone tests, because it would remove the LOCAL_RANK and then fabric would launch itself recursively.